packages/testing/src/web/mocks/form.mock.tsx

```
@@ -0,0 +1,71 @@
+ *   ...createFormMocks(),
+ * }));
+ * ```
+ */
+export function createFormMocks(): Record<
+  string,
+  (props: Record<string, unknown>) => ReactNode
+> {
+  return {
+    TextField: (props: Record<string, unknown>) => (
+      <div>
+        <label>{String(props.label ?? "")}</label>
```

Biome (2.4.6)

[error] A form label must be associated with an input.

(lint/a11y/noLabelWithoutControl)